app/pb_kits/playbook/pb_form/docs/example.yml

```diff
@@ -0,0 +1,5 @@
+examples:
+
+  rails:
+  - form_form_with: Default (Form With)
+  - form_simple_form: Default (Simple Form)
```

Comment:
Maybe my ignorance, but what's this for?

Reply:
The example / documentation that playbook shows in the web ui for the project uses this yaml file to find and render all the examples and tells it what title to use for each example.